### ISSUE TITLE
Entity without Physics Body does not respect Physics Shape's Position/Rotation

### ISSUE THREAD

**Issue by geekrelief** [OurMachinery/themachinery-public#769]
https://user-images.githubusercontent.com/27040/131052152-b2f25cd8-e943-43e0-bb40-d27eaf58ae51.mp4

**Comment by niklas-ourmachinery** [comment]
I'm confused as to what the video is supposed to show. I see the same behavior with or without a physics body, the only difference that with the body, the box reacts to the collision with the sphere where as without the body it is static (which is what you expect).

Could you explain more what the issue is here?

**Comment by geekrelief** [comment]
Sorry for the confusion.  It's been a while so I had to refamiliarize myself with the situation.  I just pulled master to confirm the issue still exists.  I recaptured the video to hopefully clarify things:

https://user-images.githubusercontent.com/27040/134452939-6ec97d0e-6f52-4225-996f-d3bfe48eeaa3.mp4

The initial collision looks correct to me since the box has a body but is displaced -1 on the y in the shape component. The box has a body so it falls and the sphere is in contact with the box's collision shape.  With body removed, the box becomes static but the sphere collides without taking the box shape's position into account.   

This is the workaround I mentioned above where I create a child entity which contains the physics shape but has its transform displaced.  

![mVRgD7nMwC](https://user-images.githubusercontent.com/27040/134453597-0ccf03b9-d7d7-46a8-a94a-1fabc66bfe38.png)

With this setup I can position the static box but have the higher.

https://user-images.githubusercontent.com/27040/134453725-fc09a92d-0b83-4be7-89d2-b27e41a11fa2.mp4

This setup with a sphere and box is just for illustrative purposes.  I discovered this issue while trying to use a plane as the static body for collision, but I wanted to use a box as its collision shape.  I wanted the box shape offset so one of the sides aligns with the plane.

https://user-images.githubusercontent.com/27040/134454035-c532a3c5-07cf-4778-aacd-48fdb12e5685.mp4
